servicecontrol/servicecontrol-instances/configuration.md

```diff
@@ -463,9 +463,14 @@ If running multiple setups of the Platform Tools (i.e. multiple versions of Serv
 
 If using [MSMQ transport](/transports/msmq) and the monitoring instance is installed on a different machine than the ServiceControl error instance, only the monitoring instance setting needs to be modified to include the machine name of the error instance in the queue address.
 
-Type: string
+| Context | Name |
+| --- | --- |
+| **Environment variable** | `LICENSECOMPONENTS_SERVICECONTROLTHROUGHPUTDADAQUEUE` |
```

Comment:
```suggestion
| **Environment variable** | `LICENSECOMPONENTS_SERVICECONTROLTHROUGHPUTDATAQUEUE` |
```

Reply:
Also is it correct that app config prefix is `LicensingComponent` and env var prefix is `LICENSECOMPONENTS` ?

Reply:
I'd say one of those is wrong - the answer to which one would be in code.

Reply:
that may be a mistake on my side